fuel-core/src/service/config.rs

```diff
@@ -64,6 +66,19 @@ impl Config {
     }
 }
 
+/// Selects the method for coordinating block
+#[derive(Clone, Debug)]
+pub enum CoordinatorConfig {
+    /// Configurable block production trigger, used for
+    /// testing and proof-of-authority networksc
```

Comment:
[nit] should `networksc` be `networksc`? I learned `iff` was a thing earlier this week so not sure if this is some abbreviation, if it is would be nice to be more clear